Sources/BrowserServicesKit/Autofill/AutofillUserScript+Email.swift

```diff
@@ -96,4 +97,20 @@ extension AutofillUserScript {
         }
     }
 
+    func emailGetUserData(_ message: AutofillMessage, _ replyHandler: @escaping MessageReplyHandler) {
+        emailDelegate?.autofillUserScriptDidRequestUserData(self) { username, alias, token, _ in
+            if let username = username, let alias = alias, let token = token {
+                replyHandler("""
+                {
+                    "userName": "\(username)",
```

Comment:
Can we encode this with an `Encodable` struct please?

Reply:
There's some in my PR: https://github.com/more-duckduckgo-org/macos-browser/pull/432 but happy to help later.

Reply:
@jonathanKingston Thanks -- updated 👍  What that what you had in mind?

Reply:
Yup thank you so much! I don't think you need the init method :)

Reply:
@jonathanKingston Even nicer ❤️